tensorrt_llm/_torch/custom_ops/cpp_custom_ops.py

```
@@ -775,15 +775,17 @@ def _(pe: torch.Tensor, nope: torch.Tensor):
         return packed, scale
 
     @torch.library.register_fake("trtllm::fp8_quantize_1x128_packed_ue8m0")
-    def _(input: torch.Tensor):
-        # Returns (fp8_e4m3 [m, k], packed_ue8m0_int32 [m, packed_sf_k])
-        # matching deep_gemm.get_mn_major_tma_aligned_packed_ue8m0_tensor's return shape.
+    def _(input: torch.Tensor, use_r128c4_layout: bool = True):
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)